PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+*Note*: Before submitting this pull request, please review our [contributing guidelines](https://github.com/tomchristie/django-rest-framework/blob/master/CONTRIBUTING.md#pull-requests).
+
+## Description
+Please describe your pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
```

Comment:
@tomchristie wasn't clear to me at first by reading that. IMHO I think the current `If it fixes a bug or resolves a feature request, be sure to link to that issue.` sounds fine. As pasting the link of the issue on the PR will make the ref link between the issue and the PR in GitHub.